SeaPkg/Library/BasePeCoffLibNegative/BasePeCoffLibNegative.c

```diff
@@ -880,6 +880,17 @@ PeCoffImageDiffValidation (
         }
 
         // Inspection of the memory attributes of the target image
+        if (ImageValidationEntryHdr->Size > sizeof(AddrInTarget)) {
+          DEBUG ((
+            DEBUG_ERROR,
+            "%a: A new cfg entry is larger a memory address!  Only pointers can have their memory attributes validated!  Address size: %x, Entry Size: %x\n",

```

Comment:
```suggestion
            "%a: A new cfg entry is larger than a memory address! Only pointers can have their memory attributes validated!  Address size: %x, Entry Size: %x\n",
```

Reply:
Probably also want to call out the addresses to help debugging (i.e. information to help understanding which entry in the cfg is violating this rule).

Reply:
I added the target address to the print statement as well as fixed the typo.